students/Thai_H/lesson2/PrintGrid.py

```diff
@@ -0,0 +1,33 @@
+def top_or_bottom(col,size):
+
+    for c in range(col):
+        print('+', end = '')
+        print( size * (' -'), end = ' ' )
+        if c == col-1:
+            print ('+')
+
+
+def sides(col,size):
+    for row_of_bar in range(size):
+        for c in range(col):
+            print ('|' , end = '')
+            print ((2*size+1) * (' '), end = '')
+            if c == col-1:
+                print ('|')
+
+
+
```

Comment:
You'll want to use blank lines sparingly - too many here.  Keep blank lines to 2 between functions, and only one between methods within classes.